IntelliTect.Analyzer/IntelliTect.Analyzer.Test/AttributesOnSeparateLinesTests.cs

```diff
@@ -291,7 +291,7 @@ static void Main()
         }
     }
 }";
-            VerifyCSharpDiagnostic(test, GetExpectedDiagnosticResult(11, 10));
+            VerifyCSharpDiagnostic(test, AttributesOnSeparateLinesTests.GetExpectedDiagnosticResult(11, 10));
```

Comment:
What is the reason for adding the class name here?

Reply:
Code Cleanup did it, ill take another look